web/styles/settings.css

```diff
@@ -1324,6 +1324,12 @@ label.preferences-radio-choice-label {
     margin-right: 1px;
 }
 
+.reactivate-user-button {
+    display: inline-flex;
+    align-items: center;
+    min-height: 35px;
```

Comment:
Generally, new sizes should always be done in `em`.

Reply:
But I think we can adjust that in a follow-up; we will eventually probably bulk-convert these.